eclipse-collections-code-generator/src/main/resources/api/set/primitiveSet.stg

```diff
@@ -69,6 +69,14 @@ public interface <name>Set extends <name>Iterable
      */
     <name>Set symmetricDifference(<name>Set set);
 
+    /**
+     * Returns {@literal true} if all the members of {@code this} are also members of {@code set}.
+     * For example, [1, 2] is a subset of [1, 2, 3], but [1, 4] is not.
+     *
+     * @since 11.0.
+     */
+    boolean isSubsetOf(<name>Set set);
```

Comment:
Hi @prathasirisha, can't we pull up the two implementations from Mutable and Immutable here?